docs/UsersGuide/source/BackgroundInfo/Introduction.rst

```diff
@@ -162,18 +175,19 @@ Community Contributions
 -------------------------
 
 The UFS community is encouraged to contribute to the development efforts of all related
-utilities, model code, and infrastructure. As described above, users can post issues in the SRW App to report bugs or to announce upcoming contributions to the code base. Additionally, users can file issues in component repositories for contributions that directly concern those repositories. Contributions to the `ufs-srweather-app <https://github.com/ufs-community/ufs-srweather-app>`__ repository should follow the guidelines contained in the `SRW App Contributor's Guide <https://github.com/ufs-community/ufs-srweather-app/wiki/Contributor's-Guide>`__. For code to be accepted into a component repository, users must follow the code management rules of that component's authoritative repository. These rules are usually outlined in the User's Guide (see :numref:`Table %s <list_of_documentation>`) or GitHub wiki for each respective repository (see :numref:`Table %s <top_level_repos>`). 
+utilities, model code, and infrastructure. As described above, users can post issues in the SRW App to report bugs or to announce upcoming contributions to the code base. Contributions to the `ufs-srweather-app <https://github.com/ufs-community/ufs-srweather-app>`__ repository should follow the guidelines contained in the `SRW App Contributor's Guide <https://github.com/ufs-community/ufs-srweather-app/wiki/Contributor's-Guide>`__. Additionally, users can file issues in component repositories for contributions that directly concern those repositories. For code to be accepted into a component repository, users must follow the code management rules of that component's authoritative repository. These rules are usually outlined in the component's User's Guide (see :numref:`Table %s <list_of_documentation>`) or GitHub wiki for each respective repository (see :numref:`Table %s <top_level_repos>`). 
 
 Future Direction
 =================
 
 Users can expect to see incremental improvements and additional capabilities in upcoming releases of the SRW Application to enhance research opportunities and support operational forecast implementations. Planned enhancements include:
 
+* Inclusion of data assimilation and forecast restart/cycling capabilities.
 * A more extensive set of supported developmental physics suites.
 * A larger number of pre-defined domains/resolutions and a *fully supported* capability to create a user-defined domain.
-* Add user-defined vertical levels (number and distribution).
-* Inclusion of data assimilation and forecast restart/cycling capabilities.
+* Incorporation of additional `Unified Workflow <https://github.com/ufs-community/workflow-tools>`__ tools. 
 
+.. COMMENT: Anything we should add to this list? Or remove?
```

Comment:
I believe that this is a good list indicating what we will be focussing on for future releases.  The only potential minor addition would be noting that JEDI will be the DA system that we will be including.  With the GSI already available, the community might wonder why the GSI hasn't been added to the SRW App.  Since we are unable to release the GSI to the community, clearing this up might be a good idea.

Reply:
@MichaelLueken Added "via JEDI" to the bullet about DA/cycling. Should I explicitly mention why we are not including GSI, or is it enough to note that we will be including DA via JEDI.

Reply:
@gspetro-NOAA Thanks!  I think adding "via JEDI" should be sufficient for DA/cycling.